projects/kit/components/input-date-time/native-date-time/native-date-time.component.ts

```diff
@@ -0,0 +1,55 @@
+import {ChangeDetectionStrategy, Component, Inject} from '@angular/core';
+import {TuiDay, TuiTime} from '@taiga-ui/cdk';
+import {TUI_TEXTFIELD_HOST} from '@taiga-ui/core';
+
+import type {TuiInputDateTimeDirective} from '../input-date-time.directive';
+
+@Component({
+    selector: 'input[tuiDateTime]',
```

Comment:
It feels like all inputs like these should be directives and styles can be applied in the corresponding control less file, since they are for internal use only.

Reply:
> Looks like this time seconds do not work even on Android. Other than that seems to be working.

 Yes, seems like no browser supports HH:MM:SS mode ( I removed the example and added a note that it only works in HH:MM mode

> It feels like all inputs like these should be directives and styles can be applied in the corresponding control less file, since they are for internal use only.

Refactored for date and date-time 👍  other controls (select, multiselect, time) should have template